index.html

```diff
@@ -121,7 +121,17 @@ <h5>Interested for more information?<br>Sign up here!</h5>
             </section>
         </main>
         <footer>
-
+            <div>
+                <span>
+                    <p>Cookies</p>
+                    <p>Privacy Policy</p>
+                </span>
+                <span>
+                    <p>Contact</p>
+                    <p>Find us</p>
+                </span>
+            </div>
+            <img src="./svg/footer-jam.svg" alt="svg of a jam with a spoon"/>
```

Comment:
I think you could omit `svg of` from the alt text :) "Jam with a spoon" is enough for screen readers :)